File: 알고리즘백준/정지관/python/2480.py
a,b,c = map(int, input().split(' '))

if(a==b==c): print(10000 + 1000*a)
elif(a == b): print(1000 + 100*a)
elif(c == b): print(1000 + 100*b)
elif(a == c): print(1000 + 100*a)
else: print(100 * ((a if a>c else c) if a>b else (b if b>c else c)))

File: 알고리즘/정지관/javascript/2480.js
let input = require("fs").readFileSync(0).toString().split(" ");

let a = parseInt(input[0]);
let b = parseInt(input[1]);
let c = parseInt(input[2]);

let res = 0;

if (a == b && b == c) {
  res = 10000 + 1000 * a;
} else if (a == b) {
  res = 1000 + 100 * a;
} else if (b == c) {
  res = 1000 + 100 * b;
} else if (c == a) {
  res = 1000 + 100 * a;
} else {
  res = Math.max(a, b, c) * 100;
}

console.log(res);
